ua-snap/rasdaman-ingest
# Preprocessing and ingest for a "wet days per year" coverage

This PR includes the preprocessing notebook and ingest recipe needed to create a Wet Days Per Year coverage for the Arctic-EDS.

The content here was used to create the `wet_days_per_year` coverage that exists on Apollo.

To test this PR:
 - Run the notebook on Atlas! All the cells should execute in sequence. You'll likely need to tweak a few env vars in the initial setup cell to match your own scratch space structure, and you can change `COPY_SOURCE` to `True` to bring the necessary source WRF data over if you don't have a copy.
 - Try to run a test copy of the ingest on Apollo from `/tmp` with all the data in a `geotiffs` directory.
